boards/atxmega-a1u-xpro/Kconfig

```diff
@@ -13,6 +13,7 @@ config BOARD_ATXMEGA_A1U_XPRO
     default y
     select CPU_MODEL_XMEGA128A1U
     select HAS_PERIPH_CPUID
+    select HAS_PERIPH_EBI
     select HAS_PERIPH_GPIO
     select HAS_PERIPH_GPIO_IRQ
     select HAS_PERIPH_NVM
```

Comment:
This all shouldn't be needed, as those are selected on the CPU level already. (Sorry that I missed this in a previous review.)

Reply:
Added `cpu/atxmega: Fix features config`